server/src/main/java/org/jetbrains/bsp/bazel/server/sync/BazelProjectMapper.kt

```diff
@@ -129,4 +135,15 @@ class BazelProjectMapper(
         with(module) {
             (sourceSet.sources + resources).map { Pair(it, label) }
         }
+
+    private fun environmentItem(target: TargetInfo): Map<String, String> {
+        val inheritedEnvs = collectInheritedEnvs(target)
+        val targetEnv = target.envMap
+        return inheritedEnvs.toMap(targetEnv)
```

Comment:
```suggestion
        return inheritedEnvs + targetEnv
```

its the reason why tests are failing - toMap puts values into the destination, and in this case it fails